modules/custom/az_media/config/install/filter.format.az_standard.yml

```diff
@@ -27,7 +28,7 @@ filters:
     weight: -10
     settings:
-      allowed_html: '<em> <strong> <s> <abbr> <address> <cite> <blockquote class=""> <br> <div class="" name="" id="" class=* data-*> <ul type class=""> <ol start type class=""> <li class=""> <dl> <dt> <dd> <h1 class="" name="" id=""> <h2 class="" name="" id=""> <h3 class="" name="" id=""> <h4 class="" name="" id=""> <h5 class="" name="" id=""> <h6 class="" name="" id=""> <sup> <sub> <table class=""> <caption> <tbody class=""> <thead class=""> <tfoot class=""> <th class="" scope="" colspan="" rowspan=""> <td class="" colspan="" rowspan=""> <tr class="" colspan="" rowspan=""> <hr class=""> <a href class="" target="" name="" id=""> <span class="" name="" id=""> <img src alt data-entity-type data-entity-uuid> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align="" data-caption data-embed-button alt title> <drupal-media data-view-mode="" data-align="" data-entity-type data-entity-uuid alt title> <p class=""> <footer class="">'
+      allowed_html: '<em> <strong> <s> <abbr> <address> <cite> <br> <div name id data-* class=""> <ol start type class=""> <li class=""> <dl> <dt> <dd> <h1 name id class=""> <h2 name id class=""> <h3 name id class=""> <h4 name id class=""> <h5 name id class=""> <h6 name id class=""> <sup> <sub> <table class=""> <caption> <tbody class=""> <thead class=""> <tfoot class=""> <th scope colspan rowspan class=""> <td colspan rowspan class=""> <tr colspan rowspan class=""> <hr class=""> <img src alt data-entity-type data-entity-uuid> <p class=""> <a href target name id href data-entity-substitution data-entity-type data-entity-uuid title class=" btn btn-red btn btn-blue btn btn-outline-red btn btn-outline-blue btn btn-outline-white btn btn-link"> <ul type class=" ul-triangles nav nav-pills"> <blockquote class=" blockquote blockquote text-center blockquote text-right blockquote blockquote-reverse text-right"> <span name id class=" text-white text-sky text-oasis text-azurite text-blue text-midnight text-dark-silver text-ash text-sans text-capitalize text-uppercase text-lowercase badge badge-red badge badge-blue badge badge-success badge badge-info badge badge-warning badge badge-danger small lead h1 h2 h3 h4 h5 h6 align-top align-middle align-bottom mt-0 mb-0"> <footer class=" blockquote-footer"> <drupal-media data-view-mode data-align data-entity-type data-entity-uuid alt title data-entity-type data-entity-uuid data-view-mode data-align data-caption alt title> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button alt title data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button data-langcode alt title data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button data-langcode alt title>'
```

Comment:
This PR looks good, but I don't believe it should change the allowed HTML tags for the standard format. Saving the format in the UI causes these tags to change from the tags we set in Quickstart.

```suggestion
      allowed_html: '<em> <strong> <s> <abbr> <address> <cite> <blockquote class=""> <br> <div class="" name="" id="" class=* data-*> <ul type class=""> <ol start type class=""> <li class=""> <dl> <dt> <dd> <h1 class="" name="" id=""> <h2 class="" name="" id=""> <h3 class="" name="" id=""> <h4 class="" name="" id=""> <h5 class="" name="" id=""> <h6 class="" name="" id=""> <sup> <sub> <table class=""> <caption> <tbody class=""> <thead class=""> <tfoot class=""> <th class="" scope="" colspan="" rowspan=""> <td class="" colspan="" rowspan=""> <tr class="" colspan="" rowspan=""> <hr class=""> <a href class="" target="" name="" id=""> <span class="" name="" id=""> <img src alt data-entity-type data-entity-uuid> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align="" data-caption data-embed-button alt title> <drupal-media data-view-mode="" data-align="" data-entity-type data-entity-uuid alt title> <p class=""> <footer class="">'
```